libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -489,20 +535,21 @@ static CK_RV prvMbedTLS_Initialize( void )
     {
         CRYPTO_Init();
         /* Initialize the entropy source and DRBG for the PKCS#11 module */
-        mbedtls_entropy_init( &xP11Context.xMbedEntropyContext );
-        mbedtls_ctr_drbg_init( &xP11Context.xMbedDrbgCtx );
+        mbedtls_entropy_init(  ( &xP11Context.xMbedEntropyContext ) );
```

Comment:
To be consistent, ```&( xP11Context.xMbedEntropyContext )``` maybe. 

Line 522, 525, ...

vs.

Line 538, 539, 541, 543